span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -205,21 +95,14 @@ public RawSpan convert(Span jaegerSpan) throws Exception {
         jaegerSpan.getTagsList().stream()
             .collect(Collectors.toMap(t -> t.getKey().toLowerCase(), t -> t, (v1, v2) -> v2));
 
-    Optional<String> maybeTenantId = this.tenantIdProvider.getTenantId(tags);
-
+    Optional<String> maybeTenantId = tenantIdHandler.getAllowedTenantId(jaegerSpan, tags);
```

Comment:
We will not need line number `94` to `108`, it has already be taken care of in `JaegerSpanPreProcessor`.